build.gradle

```
@@ -13,8 +13,33 @@ repositories {
 }
 
 dependencies {
-    implementation 'org.springframework.boot:spring-boot-starter'
+    //spring-web and test
+    implementation 'org.springframework.boot:spring-boot-starter-web'
     testImplementation 'org.springframework.boot:spring-boot-starter-test'
+
+    //lombok
+    implementation 'org.projectlombok:lombok'
+    annotationProcessor("org.projectlombok:lombok")
+
+    //jpa
+    implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
+
+    //database
+    //h2
+    implementation('com.h2database:h2')
+
+    // mustache server-template-engine
+    implementation('org.springframework.boot:spring-boot-starter-mustache')
+
+    // spring-security
+    //      소셜 로그인 등 클라이언트 입장에서 소셜 기능 구현 시 필요한 의존성입니다.
+    //      spring-security-oauth2-client와 spring-security-oauth2-jose를 기본으로 관리 해 줍니다.
+    implementation("org.springframework.boot:spring-boot-starter-oauth2-client")
+    testImplementation("org.springframework.security:spring-security-test")
```

```suggestion
    testImplementation("org.springframework.security:spring-security-test")
    implementation 'org.springframework.boot:spring-boot-starter-security'
```
bcryptEncoding을 하기 위해서 implementation spring-boot-starter-security가 필요합니다.

---

감사합니다 바로 반영해서 PR 하도록 하겠습니다.